### DISCUSSION TITLE
Lagarith frame rate issue

### DISCUSSION THREAD

**Discussion by dimar112q** [staxrip/staxrip#1560] [General]
I had audio sync issues in my project, and I noticed something strange.
My source files are AVI lagarith codec. MediaInfo shows 29.970 (30000/1001) FPS
When I add a file to StaxRip it's showing as 29.922202 fps

**Comment by Dendraspis** [comment]
AVI files are somehow "special" and many source filters have issues opening them correctly.  In `v2.44` there is an old and deprecated source filter called `AVISource` integrated, maybe you want to give him a try. Otherwise try the others to find one, that can read your source file correctly.

**Reply by dimar112q** [reply]
AVISource couldn't located a decompressor for fourcc LAGS. My next troubleshooting step is to convert AVI files to lossless mkv x264 in virtualdub2. I guess I have to give up using ancient Lagarith codec.

**Reply by Dendraspis** [reply]
I would also try the other source filters, in 99% of all cases one works.  😅

**Reply by dimar112q** [reply]
None of them worked. I ended up converting to mkv lossless x264 codec to codec in virtualdub2. Now StaxRip detects 29.970029fps which is correct.

**Reply by Dendraspis** [reply]
Interesting.  If you can create a short sample file I would be very interested to test it.

**Reply by dimar112q** [reply]
I tried creating a small sample of lagarith AVI but the end result shows a correct frame rate in StaxRip. Now I started capturing h264 VFW lossless ultra fast 4:2:2 same as capture card, with --tff interlaced command, and so far so good.
